std_msgs/Bool result
geometry_msgs/PoseStamped goal